dali/kernels/reduce/reduce_test.cc

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <random>
+#include <chrono>
+#include "dali/kernels/reduce/reduce.h"
+
+namespace dali {
+namespace kernels {
+
+using perfclock = std::chrono::high_resolution_clock;
```

Comment:
Is it only for debugging? Maybe perf test should be moved to the benchmark?

Reply:
Sort of. It doesn't spam the output too much and when we have time (I wonder when, hehe) to optimize, then it's already there...